db/schema.rb

```diff
@@ -33,8 +33,10 @@
   create_table "loans", force: :cascade do |t|
     t.integer  "project_id"
     t.integer  "order_id"
-    t.datetime "created_at", null: false
-    t.datetime "updated_at", null: false
+    t.datetime "created_at",     null: false
+    t.datetime "updated_at",     null: false
+    t.integer  "quantity"
+    t.integer  "line_item_cost"
```

Comment:
Yes, this was my concern on @oorion's pull as well. My understanding is that is it always a good idea to use migrations to change the schema, rather than changing the schema.rb directly. Thoughts?